src/components/shared/DataTypeSelect.vue

```diff
@@ -1,56 +1,83 @@
 <template>
-  <traction-select
-    id="data-type"
-    :model-value="value"
-    data-attribute="data-type-list"
-    :options="dataTypes"
-    @update:model-value="handleInput"
-  ></traction-select>
+  <traction-field-group label="Data Type" label-for="data-type" layout="spacious">
+    <traction-select
+      id="data-type"
+      :model-value="dataType"
+      data-attribute="data-type-list"
+      :options="dataTypes"
+      @update:model-value="handleInput"
+    ></traction-select
+  ></traction-field-group>
 </template>
 
 <script>
 import useSWRV from 'swrv'
 import { filterByAttribute, mapAttribute } from '@/api/JsonApi'
 
+const encode = (value) => (value === undefined ? 'basecalls and raw data' : value)
+const decode = (value) => (value === '' ? null : value)
+
 export default {
   name: 'DataTypeSelect',
   props: {
     pipeline: {
-      // Filter to only list data_types for a given pipeline, leave null
-      // to apply no filters
       type: String,
       required: false,
       default: null,
     },
-    value: {
-      // The data type, we use value to allow us to simply bind it with v-model
+    modelValue: {
       type: String,
       default: undefined,
     },
   },
-  emits: ['input'],
+  emits: ['update:modelValue'],
   setup() {
     const baseURL = import.meta.env.VITE_TRACTION_BASE_URL
     const { data: remoteDataTypes } = useSWRV(
       `${baseURL}/v1/data_types?fields[data_types]=name,pipeline`,
     )
     return { remoteDataTypes }
   },
+  data() {
+    return {
+      dataType: encode(this.modelValue),
+    }
+  },
   computed: {
+    // Same approach as src/components/shared/LibraryTypeSelect.vue
     filters: ({ pipeline }) => (pipeline ? { pipeline } : {}),
```

Comment:
I think you need to add the filter back into here for 'basecalls and raw data'

Reply:
oh no..